test-suite/consensus-tests/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
```

Comment:
A README for the crate would be appreciated. Despite the crate not being published, README could still help understand the purpose and contents of the crate.